easybuild/easyconfigs/c/CPMD/CPMD-4.1-CrayIntel-17.08-pat-645-nogpu.eb

```diff
@@ -0,0 +1,43 @@
+# contributed by Andreas Jocksch, Luca Marsella and Guilherme Peretti Pezzi (CSCS)
+easyblock = 'MakeCp'
+
+name = 'CPMD'
+version = '4.1'
+patversion = '645-nogpu'
+versionsuffix = '-pat-%s' %(patversion)
+
+homepage = 'http://www.cpmd.org/'
+description = """CPMD The CPMD code is a parallelized plane wave / pseudopotential
+implementation of Density Functional Theory, particularly designed for ab-initio molecular dynamics. """
+
+toolchain = {'name': 'CrayIntel', 'version': '17.08'}
+toolchainopts = {'usempi': True}
+
+sources = ['/apps/common/UES/easybuild/sources/c/CPMD/cpmd-v%s.tar.gz' %version]
+
+patches = [('CRAY-XE6-INTEL-MKL', 'CPMD/configure')]
+
+dependencies = [('fftw/3.3.4.10', EXTERNAL_MODULE)]
+
+skipsteps = ['configure']
+
+common_prebuildopts = './configure.sh PREFIX=%(installdir)s CRAY-XE6-INTEL-MKL &&'
```

Comment:
I don't see why you need to define this as a variable, couldn't you add this string directly to prebuildopts?

Reply:
If we want to keep the OpenMP option, which might work for a next release, the variable is used twice. Thus it is efficient.